itez/beneficiary/uuid.py

```diff
@@ -0,0 +1,11 @@
+import uuid
+
+
+def generate_id():
+    """
+     this function generate a 7 digit code for agent
+    """
+    uuid_one = uuid.uuid4()
+    print(uuid_one)
+    agent_code = str(uuid_one)[1:8].upper()
+    return (agent_code)
```

Comment:
we are looking to have both the uuid and agent code from this utitility
why are we returning on the  agent code?

I thought I wrote demo code  for you already?

Reply:
now returning uuid as well